src/test/webServer.ts

```diff
@@ -103,31 +103,23 @@ export class WebServer {
   }
 }
 
-async function canBindPort(port: number): Promise<boolean> {
+async function isPortUsed(port: number): Promise<boolean> {
```

Comment:
Replaced `canBindPort` by `isPortUsed` using the code in `waitForSocket`, which now uses this function along with the above private `WebServer._startProcess`.